Library/Homebrew/cmd/man.rb

```diff
@@ -11,26 +11,14 @@ def man
     raise UsageError unless ARGV.named.empty?
 
     if ARGV.flag? "--link"
-      link_man_pages
+      odie "`brew man --link` is now done automatically by `brew update`."
```

Comment:
I think we could also drop this completely. `brew man` has never been documented (as far as I'm aware of) and its `--link` option is probably even more obscure and unknown.